### ISSUE TITLE
updating the train.py

### ISSUE THREAD

**Issue by ShunZuo-AI** [inging550/AlexNet-Pytorch-#1]
There is a problem in train.py that the best_test_loss is 0.0 and the trained model cannot be saved automatically,it should be given an large number

**Comment by inging550** [comment]
You are right, it's suitable that the best_test_loss init in a large number like 20
